src/UserSpaceInstrumentation/AccessTraceesMemoryTest.cpp

```diff
@@ -172,18 +172,18 @@ TEST(AccessTraceesMemoryTest, WriteFailures) {
 
 TEST(AccessTraceesMemoryTest, ReadWriteRestore) {
   pid_t pid = fork();
-  ASSERT_TRUE(pid != -1);
+  CHECK(pid != -1);
   if (pid == 0) {
     // Child just runs an endless loop.
     while (true) {
     }
   }
 
   // Stop the child process using our tooling.
-  ASSERT_TRUE(AttachAndStopProcess(pid).has_value());
+  CHECK(!AttachAndStopProcess(pid).has_error());
```

Comment:
Are you sure we want to loose error message here? I feel like this making it harder to diagnose broken tests.